actors/builtin/multisig/multisig_actor.go

```diff
@@ -279,6 +297,32 @@ func (a Actor) ChangeNumApprovalsThreshold(rt vmr.Runtime, params *ChangeNumAppr
 	return nil
 }
 
+func (ahd *ApprovalHashData) Serialize() ([]byte, error) {
+	buf := new(bytes.Buffer)
+	if err := ahd.MarshalCBOR(buf); err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+func (a Actor) getApprovalHash(rt vmr.Runtime, txn Transaction) [32]byte {
```

Comment:
naming inconsistency following from `ApprovalHashData`
this is the hash of the proposal